src/private/files.ts

```diff
@@ -21,18 +21,35 @@ export namespace files {
     Sharefile = 'SHAREFILE',
     GoogleDrive = 'GOOGLEDRIVE',
     Egnyte = 'EGNYTE',
+    SharePoint = 'SharePoint',
+  }
+  interface IWopiThumbnail {
+    size: number;
+    url: string;
+  }
+
+  interface IWopiService {
+    name: string;
+    description: string;
+    thumbnails: IWopiThumbnail[];
+  }
+
+  export interface IExternalProvider extends IWopiService {
+    providerType: CloudStorageProviderType;
+    providerCode: CloudStorageProvider;
   }
 
-  /**
-   * @private
```

Comment:
Did you intentionally remove the `@private` comment? This will cause the interfaces/enums to be added to our public docs here which might not be what we want since these APIs won't work for 3P apps:
https://docs.microsoft.com/en-us/javascript/api/overview/msteams-client?view=msteams-client-js-latest